slacker.py

```diff
@@ -241,22 +255,17 @@ def post_message(self, channel, message, message_type=None):
             channel = channel[1:]
 
         post_data = {
-            'token': self.token,
+            'token': self.bot_token,
             'channel': channel,
             'text': message.encode('utf-8')
         }
 
-        bot_name = self.config.bot_name
-        bot_avatar_url = self.config.bot_avatar_url
-        if bot_name or bot_avatar_url:
-            post_data['as_user'] = False
-            if bot_name:
-                post_data['username'] = bot_name
-            if bot_avatar_url:
-                post_data['icon_url'] = bot_avatar_url
-
         if message_type:
-            post_data['attachments'] = json.dumps([{'fallback': message_type}], encoding='utf-8')
```

Comment:
Is this now the default?

Reply:
yes, I think it is a side effect of running on python 3